bug-bounty-program.md

```diff
@@ -4,36 +4,34 @@ The ENS bug bounty program rewards anyone who finds a bug in covered ENS smart c
 
 ## Rules and Rewards
 
-* Issues that have already been submitted by another user or are already known to the ENS team are not eligible for bounty rewards.
-* Public disclosure of a vulnerability makes it ineligible for a bounty. This includes exploiting the bug on mainnet or any public test network.
+* Reward sizes are _guided_ by the rules below, but are in the end, determined at the sole discretion of the ENS team
 * The ENS team, employees and all other people paid by ENS project, directly or indirectly, are not eligible for rewards.
-* Only the smart contracts listed below are eligible for rewards. Websites and other infrastructure are not covered by the bounty program.
 * The ENS bounty program considers a number of variables in determining rewards. Determinations of eligibility, score and all terms related to an award are at the sole and final discretion of the ENS team.
+* Issues that have already been submitted by another user or are already known to the ENS team are not eligible for bounty rewards.
+* Public disclosure of a vulnerability makes it ineligible for a bounty. This includes exploiting the bug on mainnet or any public test network.
+* The bounty program applies to security vulnerabilities, not inconsistencies in the normalization function. 
+* As of 31 July 2022 the normalization function is under active revision at the moment and is not eligible for a bounty.
```

Comment:
The normalisation function was already not covered; it's not a smart contract, and for the most part issues with it aren't security vulnerabilities.